infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -341,6 +341,24 @@ func CppifyHeaderIncludes(contents string) (string, error) {
 	return contents, nil
 }
 
+func AppendStringToFile(filepath, new_content string) error {
+	// Appends |new_content| to the content of |filepath|.
+	file, err := os.OpenFile(filepath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	_, err = file.WriteString(new_content)
+	return err
+}
+
+func RecordError(errstr string) {
```

Comment:
nit: Maybe change this to WriteStdErrOut and take both as input, write both to stdout/stderr and record stderr.

Reply:
Yep, good point thanks!
Fixed : )